changelogs/22.04

```diff
@@ -0,0 +1,52 @@
+* Pop!_OS is now based on Ubuntu 22.04.
+
+* COSMIC
+** GNOME 42 support has been added for COSMIC.
+** The Shell Theme for GNOME 42 has been updated.
+** Icons are now SVG-based instead of PNG-based.
+** Dark and light modes can have independent backgrounds.
+** Settings for Desktop Options, Background, Appearance, Dock, and Workspaces are now searchable in Launcher.
+** The Pop robot face has been added as the default avatar.
+** Improvements to cosmic-workspaces:
+    ** An issue with the location of the Workspace picker on secondary displays has been fixed.
+    ** An issue with the scaling of the Workspace picker on displays of varying resolutions has been fixed.
+    ** Performance has been improved.
+    ** An issue with alignment on HiDPI displays has been fixed.
+    ** The OSD picker is now smaller.
+** COSMIC logging has been limited, resulting in increased system performance.
+
+* Pop!_OS System Updater
+** New upgrade and options for installing upgrades are now available in Settings > OS Upgrade and Recovery.
+    ** To reduce distractions, update notifications are now set to show weekly and can be set to daily or monthly.
+    ** Upgrades can be scheduled and installed automatically (includes Debian, Flatpak, and *nix packages).
```

Comment:
Again, we might want to stay consistent with calling them "updates" instead of "upgrades" for this feature, but not necessarily an issue if we don't want to go that direction.